creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -990,34 +994,31 @@ class SnapbackSM {
     }
 
     const startTimeMs = Date.now()
-    const endTimeMs = startTimeMs + MaxSyncMonitoringDurationInMs
+    const maxMonitoringTimeMs = startTimeMs + MaxSyncMonitoringDurationInMs
 
-    let additionalSyncRequired = true
-    let maxExportRangeExceeded = false
-    while (Date.now() < endTimeMs) {
+    let secondaryCaughtUp = false
+    let secondaryProgressed = false
+    let initialSecondaryClock = null
+    let finalSecondaryClock = null
+    while (Date.now() < maxMonitoringTimeMs) {
       try {
         const clockStatusResp = await axios(clockStatusRequestParams)
```

Comment:
should we be using the `CreatorNode.getClockValue()` fn here instead?

Reply:
you right, done